docs/user-guide/install-configure-zos-extensions.md

```diff
@@ -50,69 +32,43 @@ Execute the command from z/OS USS. Use the following command line parameters:
 
   (Boolean, Optional) Enable the most detail trace level logging. This will help on troubleshooting issues.
 
-**Examples:**
-
-The following command installs the `my-zowe-component-1.2.3.pax` into `/global/zowe/extensions` which is defined as `zowe.extensionDirectory` in `/path/to/my/zowe.yaml`.
-
-```
-$ zwe components install -o /path/to/my-zowe-component-1.2.3.pax -c /path/to/my/zowe.yaml
-```
-
 ## Enable and disable component
 
-Zowe ships [`zwe components enable`](../appendix/zwe_server_command_reference/zwe/components/zwe-components-enable.md) and [`zwe components disable`](../appendix/zwe_server_command_reference/zwe/components/zwe-components-disable.md) commands to help you enable and disable Zowe server component (extension). In order to be compatible with these commands, components must follow [Zowe server component package format standard](packaging-zos-extensions.md#zowe-server-component-package-format).
+Zowe ships [`zwe components enable`](../appendix/zwe_server_command_reference/zwe/components/zwe-components-enable.md) and [`zwe components disable`](../appendix/zwe_server_command_reference/zwe/components/zwe-components-disable.md) commands to help you enable and disable Zowe server component (extension). In order to be compatible with these commands, components must follow [Zowe server component package format standard](../extend/packaging-zos-extensions.md#zowe-server-component-package-format).
```

Comment:
Please do a text search in this document for all occurrences of `](` (?) and see if any other link will need to be updated as a result of this file's move

Reply:
the only one i cant handle in this pr is the zwe help one because thats auto-generated from zowe-install-packaging.